### DISCUSSION TITLE
[Review pull requests] Course feedback survey!

### DISCUSSION THREAD

**Discussion by nguyenalex836** [skills/.github#549] [Review pull requests]
Thank you for completing this course! ✨ We are delighted to have you as a user, and are committed to providing the best possible experience through GitHub Skills.

Please take a moment to answer as many (or as few) questions as possible regarding the course you have just completed.

1. What part of the course did you enjoy the most?
2. What level of experience do you have with GitHub?
3. What’s something new you learned that you didn’t know before?
4. What’s something about the course that you didn’t enjoy as much?
5. What would you like to learn more about regarding GitHub?
6. What are you planning to use GitHub for?
7. How likely are you to recommend this course to a friend or colleague?

Your feedback is invaluable as we endeavor to enhance GitHub Skills for all GitHub users 💛

**Comment by yOOs-dz** [comment]
Hello all, 
I would like to thank you about this insightful course that I enjoyed it all. I have to do too much practice on real projects to well understand. Some parts remain for me too theoretical and need an effort to get their applicability.
Many  thanks a second time.

**Comment by julioaranajr** [comment]
> What part of the course did you enjoy the most?
- assigning pull requests for review. 💁🏼 
- review on a pull request. 🧑‍🏫 
- suggesting changes to a pull request. 🧑‍🦱

**Reply by nguyenalex836** [reply]
@julioaranajr Thank you very much for this feedback! ✨ 🚀

**Comment by yashksaini-coder** [comment]
- [x] **Assigning a pull request for review:** 💁🏼
- [x] **Reviewing a pull request:** 🧑‍🏫
- [x] **Suggesting changes to a pull request:** 🧑‍🦱

**Reply by Mortiemi** [reply]
Thank you very much for your help. And I apologise for the trouble I created.😢

**Comment by Valentiramirez** [comment]
Hola es que necesito ayuda con el curso de GitHub y es que ya lo termine pero en la página de GitHub Educación no me aparece que lo finalize no sé si hay que enviarlo a una página porque ya hice todos los pasos. Si alguien me puede ayudar

**Comment by edmonsantos** [comment]
Thank You for that, id would love to know more all about this  especially from a newbie trying to find my first job.. Thanks.

**Comment by Ripididip** [comment]
> Thank you for completing this course! ✨ We are delighted to have you as a user, and are committed to providing the best possible experience through GitHub Skills.
> 
> Please take a moment to answer as many (or as few) questions as possible regarding the course you have just completed.
> 
> 1. What part of the course did you enjoy the most?
> 2. What level of experience do you have with GitHub?
> 3. What’s something new you learned that you didn’t know before?
> 4. What’s something about the course that you didn’t enjoy as much?
> 5. What would you like to learn more about regarding GitHub?
> 6. What are you planning to use GitHub for?
> 7. How likely are you to recommend this course to a friend or colleague?
> 
> Your feedback is invaluable as we endeavor to enhance GitHub Skills for all GitHub users 💛

**Comment by egoshina08** [comment]
К сожалению, я не могу понять, что нужно сделать с пунктом, который вы не привели. Пожалуйста, переформулируйте запрос, если он остался непонятным.

**Reply by ButtButts** [reply]
> К сожалению, я не могу понять, что нужно сделать с пунктом, который вы не привели. Пожалуйста, переформулируйте запрос, если он остался непонятным.

[change_management_whitepaper_workforce_adoption_journey_with_chrome_enterprise.pdf](https://github.com/user-attachments/files/19198422/change_management_whitepaper_workforce_adoption_journey_with_chrome_enterprise.pdf)
![IMG_0169](https://github.com/user-attachments/assets/340bb1d3-dc49-4ff1-836a-3c5ff058f07c)
![IMG_0812](https://github.com/user-attachments/assets/141263a9-5c2d-440c-80fa-dc0edb7251f3)
![IMG_0802](https://github.com/user-attachments/assets/3a9d7b9f-f2f5-44af-9dd6-ab4d8b66718a)
![IMG_0800](https://github.com/user-attachments/assets/dd09a5d0-a47c-4f18-b565-fc5e849968c8)
![IMG_0792](https://github.com/user-attachments/assets/7a4618fa-a4d9-4a7a-8d2b-bd55fa02f94e)
![IMG_0787](https://github.com/user-attachments/assets/adb2b313-0609-466a-9491-9ac06348fe9a)
![IMG_0786](https://github.com/user-attachments/assets/e50e96dc-6f26-40e3-aa72-7b49ac2fbf78)
![IMG_0183](https://github.com/user-attachments/assets/d448fe86-6f8a-42af-80a6-970b6348cd09)
![IMG_0782](https://github.com/user-attachments/assets/a55b780a-9c7b-4932-87f6-987c5b29a717)
![IMG_0788](https://github.com/user-attachments/assets/6f644e1c-b103-4956-add9-8a13a29b2477)
![IMG_0799](https://github.com/user-attachments/assets/9e143887-2099-4cd4-9e5b-37d3d8318e21)
![IMG_0800](https://github.com/user-attachments/assets/ddfe042a-6734-490b-901f-29923b51209d)

**Comment by egoshina08** [comment]
Р

**Comment by Altamash-dev** [comment]
1) The 20 sec refresh idea which updates the courses, works like charm.

2) beginner , smooth sailing so far.

3) "commit" - Hats off .

p.s : course in progress (Intro to Github) .

**Comment by NgocChauNguyen-CPP** [comment]
1. What I liked most about the course was that it had multiple concrete milestones and clear instructions. I liked the green color of the progress update--it gave me indication that I was on the right track. I also like Mona, who checked my work and unlocked new steps for me. Having the tutorials unlock one step at a time helped me stay calm and focus, as I did not feel overwhelm by the amount of total steps/ the whole things I need to make. 
2. I am a newbie with GitHub. 
3. About push request and merge. Thank you for embedding the links that lead to the glossary. It was really helpful. ChatGPT also helped me a long the way as well. 
4. I like everything about the course. 
5. I would like to learn more about the standard features of GitHub that students should be well-versed in before entering the job market. I also would love to learn how to organize my repositories, write READMEs well, embed cool features on them, write a write-up, and use GitHub as my professional platform. 
6. I plan to use GitHub as my portfolio and my playground to practice my skills. 
7. I would love to recommend the course to everyone. 10/10
8. Thank you for putting the course together. Appreciate the work.

**Comment by mhihmi** [comment]
1. The part about suggesting changes on a pull request was nice ! 
2. I'm a user of github since few years now, for professional or personal projects.
3. I didnt know about this button "file changes" at the bottom... I think it isn't enough visible. Usually I don't check file changes this way, clicking on this one...
4. I didn't like the fact that we had to merge on main branch..., it's not a good practice : you really should write a ☠️ BIG Warning 😈 about that in this course 😇
5. Using perfectly Dependabots
6. Working & personal projects, contribution to Open source projects.
7. This course should be included in every school who use github, for students, beginners, and even more advanced to remind stuff we can forget..  10/10 on recommendation.

Big thanks for your work. I'm really curious about the script you used in Action for this one ! 😋

**Comment by dutrevis** [comment]
Hi! I bring here some suggestions to enhance the basic knowledge provided by the Pull Request module's exercise:
1. At the branches explanation, some different concepts of using branches to merge changes could be shown, aside merging into the main branch, like the release branch strategy.
2. Solving conflicts could be addressed, as it is a huge and obscure blocker for newbies when applying changes;
3. At the merge, squash and rebase commits explanation, a single image would help a lot to visualize the concepts, like the one below: ![commit concepts](https://substackcdn.com/image/fetch/$s_!sKSs!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F1ed5f5ba-940f-4693-9303-dd9297010fc2_1543x576.jpeg)

**Comment by muyiboy** [comment]
> Hi! I bring here some suggestions to enhance the basic knowledge provided by the Pull Request module's exercise:
> 1. At the branches explanation, some different concepts of using branches to merge changes could be shown, aside merging into the main branch, like the release branch strategy.
> 2. Solving conflicts could be addressed, as it is a huge and obscure blocker for newbies when applying changes;
> 3. At the merge, squash and rebase commits explanation, a single image would help a lot to visualize the concepts, like the one below: ![commit concepts](https://substackcdn.com/image/fetch/$s_!sKSs!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F1ed5f5ba-940f-4693-9303-dd9297010fc2_1543x576.jpeg)
>

**Reply by 6990571434** [reply]
hi
